### DISCUSSION TITLE
Generating coastline tiles

### DISCUSSION THREAD

**Discussion by sombell** [systemed/tilemaker#737] [Q&A] [answered]
More than possible I've missed something obvious, so apologies if this is just my own stupidity.

I'm attempting to generate the basic coastline tile as shown in the RUNNING.md documentation. Specifically running this command:

```
tilemaker --output coastline.mbtiles \
          --bbox -180,-85,180,85 \
          --process resources/process-coastline.lua \
          --config resources/config-coastline.json
```

I've tried with the 3.0.0 binary, and building myself from source.

In both cases I get "No source .osm.pbf file supplied". 

I've seen that running tilemaker without an .osm.pbf file is now supported, but that a "bounding_box" must be provided. Using the supplied resources/config-coastline.json, and modifying it to include an explicit "bounding_box".

Any help appreciated

**Comment by systemed** [comment, marked as answer]
Are you sure you don't have an old version of tilemaker in your path? That string hasn't been in the source code for a while.

**Reply by sombell** [reply]
Definitely not sure! (but now even more puzzled)

I downloaded the binary (ubuntu.22.04) and the source (tar.gz) from this page:

https://github.com/systemed/tilemaker/releases/tag/v3.0.0

Is this not the correct link?

EDIT: files show modified date of 14:56 15 Jan 2024.
I'm guessing this means 3.0.0 is in fact an old version. (or that somehow what I have isnt 3.0.0)

**Comment by sombell** [comment]
Bingo! Thank you for the clue. As I suspected it was either something i missed, or an erroneous assumption.
